### DISCUSSION TITLE
GoogleAdServices

### DISCUSSION THREAD

**Discussion by lz3ai** [AdguardTeam/AdGuardHome#7133] [Q&A]
Hello,
I use AdGuard Home as DNS for my local network at home. If someone is searching for something with Google and decides to click on a sponsored link, then they are blocked. If it copies the visible link of the searched site, it opens. But if he clicks on it, as far as I can see a redirect to https://www.googleadservices.com/pagead/-something-is-blah-blah... apparently to account for the click on the ad link. But the effect of using AdGuard Home as DNS is to block this action. I tried inserting the googleadservices.com domain as always allowed in the local filtering rules, but that didn't help. I would be grateful if anyone could help with this situation. I personally don't mind copying and pasting the visible link and getting the job done, but I have a hard time convincing others at home that using this DNS is better if they can't work the traditional way.

**Comment by lbeasti** [comment]
Hi,
I have the same problem. Even deactivating the protection does not solve it. Only the change of the dns server lets me open sponsered links. Why are they blocked though put on the whitelist?
Regards
Manuel

**Comment by lbeasti** [comment]
hello, anybody at home?

**Comment by Dynamic5912** [comment]
> Hi,
> 
> I have the same problem. Even deactivating the protection does not solve it. Only the change of the dns server lets me open sponsered links. Why are they blocked though put on the whitelist?
> 
> Regards
> 
> Manuel

What DNS server/s are you using currently?

Have you checked the Query Log after it's blocked to confirm you're whitelisting the correct DNS names?

**Reply by lbeasti** [reply]
I'm using 94.140.14.14 and .15
In the query log I can see that it's forwarded.
It seems to be a problem in combination with the chrome browser on android. On a windows pc I can open the links

**Comment by lz3ai** [comment]
I use Windows 11 on PC with Chrome, Brave and Edge - there is no differencies. I have Firefox but I can't try it. I will.

**Comment by lbeasti** [comment]
I have to correct myself, it also happens with firefox. And also with ad.doubleclick.net though it's put on the whitelist and logged as forwarded.
it's really annoying, i have to consider turning adguard off

**Comment by julienval** [comment]
I face the exact same problem. Workaround is to disable all filters/rules *AND* the AdGuard as a whole (both are needed which is not understandable for me).

**Reply by Dynamic5912** [reply]
What shows in the logs as blocking?

**Reply by julienval** [reply]
All requests are marked as "treated". I see some requests coming from the same client on bot ipv4 and ipv6
![Capture d'écran 2024-09-29 165839](https://github.com/user-attachments/assets/f7d83570-dd08-402b-83f5-fbef706b1673)

**Reply by Dynamic5912** [reply]
Those are not google ad services urls like in the original posters comment

**Reply by julienval** [reply]
Indeed, but it is what I see in the logs... and here is a screenshot of the error I see from my phone (the client here)
![share_136801397736439773](https://github.com/user-attachments/assets/98db48e1-a205-40c0-99ad-69bf58badedd)

**Reply by julienval** [reply]
If I switch to the 5G netwrok and update the page, everything works as expected. If I disable all AdGuard rules/filters and AdGuard Home itself, it also works.

**Reply by Dynamic5912** [reply]
**googleadservices** will show in the logs as blocked as that's the domain being visited

**Reply by danmo24** [reply]
I tried also a lot of different settings without luck.
Here a solution is shown but how i understood not for the selfhosted Adguard:
https://www.reddit.com/r/Adguard/comments/w0922m/comment/is64s5b/?utm_source=share&utm_medium=mweb3x&utm_name=mweb3xcss&utm_term=1&utm_content=share_button

**Reply by jpospi** [reply]
I am facing the same problem. Are there any workarounds or solutions yet?

**Reply by Dynamic5912** [reply]
> I am facing the same problem. Are there any workarounds or solutions yet?

I what's your setup?

Upstream DNS, Blocklists etc.

**Reply by jpospi** [reply]
AdGuard Home on my Synology with Docker  
Upstream DNS: 8.8.8.8; 1.1.1.1
pretty much everything on default
Filter: AdGuard DNS filter; AdAway Default Blocklist

**Reply by Dynamic5912** [reply]
> AdGuard Home on my Synology with Docker  
> Upstream DNS: 8.8.8.8; 1.1.1.1
> pretty much everything on default
> Filter: AdGuard DNS filter; AdAway Default Blocklist

Googleadservives domain is blocked by AdGuard DNS Filter list...

**Reply by danmo24** [reply]
![Screenshot_20250121-093554](https://github.com/user-attachments/assets/dfd14caf-122b-469e-939b-a98ea4d3a7f5)
![Screenshot_20250121-093519](https://github.com/user-attachments/assets/728620f7-9ad9-465a-8ea1-07e5850f9f00)
My setup works now for most GoogleAd URLs 
I have whitelisted some Domains, see attachment:
@@||googleadservices.com^
@@||ad.doubleclick.net^

**Reply by Dynamic5912** [reply]
Unblocking works for me:

![image](https://github.com/user-attachments/assets/cc2326e6-abdf-4090-b829-db8ab03b6e37)
